### PR TITLE
Feat/delete modal

### PR DESCRIPTION
**Pull request by raphael-arce**
added a confirm delete account dialog instead of the browser alert

### CODE REVIEW

**Comment by raphael-arce** [comment]
@aeschi  could you please check the buttons here and the design in general? thx!

File: src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			setSearchParams(new URLSearchParams());
+		} catch (error) {
+			handleError(i18n.common.defaultErrorMessage, error);
+		}
+	}, []);
+
+	return (
+		<dialog
+			id={"delete-account-dialog"}
+			className="pt-9 pb-12 px-14 bg-white shadow rounded-2xl backdrop:backdrop-blur"
```
Comment (aeschi) [inline src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx:27]: there is a bit too much white space with these padding settings. I checked again on figma, and I would change it to: pt-10 pb-8 px-12

File: src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		>
+			<p className="font-semibold text-xl">
+				{
+					useI18nStore.getState().i18n().navbar.profile.settings
+						.deleteAccountConfirm
+				}
+			</p>
+			<div className="flex flex-col sm:flex-row justify-between pt-12">
```
Comment (aeschi) [inline src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx:35]: regarding comment above:  I would change pt12 to pt-8

**Review by aeschi — CHANGES_REQUESTED** [review]
a few design changes, otherwise LGTM

File: src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx (hunk 1 above)
Comment (raphael-arce) [reply src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx]: will do 👍

File: src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx (hunk 2 above)
Comment (raphael-arce) [reply src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx]: will do 👍

**Pull request merged**